q_learning.py

```diff
@@ -197,23 +184,38 @@ def plotRewards(ax, rewards, stderr, label):
   ax.plot(rewards, label=label)
   ax.fill_between(range(rewards.shape[0]), low_ci, high_ci, alpha=0.4)
 
-fig = plt.figure()
-ax = plt.axes()
+# fig = plt.figure()
+# ax = plt.axes()
 
-env = GridWorld([30, 30], 400)
+# def main():
+env = GridWorld([5, 5], 400)
 
 # Optimal for riverswim, doesn't make sense on gridworld
 # (rewards, stderr) = averageOverRuns(Optimal, env, 20)
 # plotRewards(ax, rewards, stderr, 'Optimal')
+#
+# (rewards, stderr) = averageOverRuns(Q, env, 1)
+# plotRewards(ax, rewards, stderr, 'Q epsilon=0.1')
 
 # (rewards, stderr) = averageOverRuns(Q, env, 1)
 # plotRewards(ax, rewards, stderr, 'Q epsilon=0.1')
 
 (rewards, stderr) = averageOverRuns(QRewardValueFunction, env, 5)
 plotRewards(ax, rewards, stderr, 'QReward value-function')
 
-plt.legend()
-plt.show()
+# plt.legend()
+# plt.show()
+
+
+
+def parse_args():
+    parser = argparse.ArgumentParser("Reinforcement Learning experiments for multiagent environments")
```

Comment:
I'm getting errors saying that `argparse` is not defined. Does this need to be imported?